.circleci/regtest-config.yml

```diff
@@ -33,36 +33,18 @@ jobs:
     resource_class: 2xlarge
     steps:
       - checkout
-      - restore_cache:
-          name: "Restore engine cache"
-          key: engine-regtest-build-{{ arch }}-
-      - run:
-          name: "Checkout devtools"
-          command: git clone https://github.com/secretflow/devtools.git
-      - run:
-          name: Decompress cache
-          command: sh devtools/decompress-build-cache.sh engine_regtest_cache
       - run:
           name: "Build binary"
           command: |
             set +e 
             mkdir -p image
             cp docker/scql.Dockerfile image
             docker run -d -it --name scql-dev -v $(pwd):/home/admin/dev/ -v ~/.cache/engine_regtest_cache:/root/.cache/engine_regtest_cache -w /home/admin/dev --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --cap-add=NET_ADMIN --privileged=true secretflow/release-ci:latest
-            docker exec -it scql-dev bash -c "make && bazel build //engine/... -c opt --ui_event_filters=-info,-debug,-warning --disk_cache=~/.cache/engine_regtest_cache" 
+            docker exec -it scql-dev bash -c "make && bazel build //engine/... -c opt --ui_event_filters=-info,-debug,-warning" 
```

Comment:
1.why the gcs not work for regtest?
2. dir in line42: ~/.cache/engine_regtest_cache can be cleaned.

Reply:
> 1.why the gcs not work for regtest? 2. dir in line42: ~/.cache/engine_regtest_cache can be cleaned.

It works, just we do not provide cache for build based on release-ci at this point.